.goreleaser.yml

```diff
@@ -20,6 +20,8 @@ builds:
         goarch: arm
     env:
       - CGO_ENABLED=0
+    ldflags:
+      - -s -w -X github.com/controlplaneio/kubesec/v2/cmd.version={{.Version}} -X github.com/controlplaneio/kubesec/v2/cmd.commit={{.Commit}} -X github.com/controlplaneio/kubesec/v2/cmd.date={{.Date}}
```

Comment:
`.Commit` is deprecated in favor of `.FullCommit` as per doc: https://goreleaser.com/customization/templates/